### DISCUSSION TITLE
whats the purpose of the server dropdown on the docs page and how to configure it

### DISCUSSION THREAD

**Discussion by bluebrown** [fastapi/fastapi#4291] [Questions] [answered]
Hi, I noticed this dropdown on my docs page.  What is its purpose, and how do I configure it?

![dropdown](https://user-images.githubusercontent.com/39703898/146408924-e569ebd5-8b7d-46f5-aa4a-166c665d0d15.png)

 It only started showing up after mounting a sub app, I think. But mounting 2 subapps did not result in them being both in the dropdown.

My subapps are mounted like 

```python
root_app.mount("/v1", v1)
root_app.mount("/v2", v2)
```

**Comment by YuriiMotov** [comment, marked as answer]
Docs: https://fastapi.tiangolo.com/advanced/behind-a-proxy/#check-the-docs-ui
